viewer/viewer.app.js

```diff
@@ -31,11 +31,11 @@
         // should we allow for improper URLs here?
         // what if there are 2 filters and the id filter is the second one.
         // Is that improper or should it be parsed and ignore the other filter?
-        angular.forEach(context.filters, function(filter) {
-            if(filter.name.toLowerCase() === "id") {
-                context.imageID = filter.value;
-            }
-        });
+        if (context.filter.type === "BinaryPredicate" &&
```

Comment:
Tiny nit here but I see `context.filter` accessed multiple times and it's slightly better to just store that in a variable for quicker access.